changelog.txt

```diff
@@ -1258,7 +1258,7 @@
   * more BATS tests
   * gating: clean /go/bin to install fresh tools
   * make lint: enable gocritic
-  * linter: blacklist linters instead of whitelisting
+  * linter: denylist linters instead of allowlist
```

Comment:
Remove this change, it is in the past and will be in the git changelog, so we should stay consistent.